paper-toast.html

```diff
@@ -41,12 +42,14 @@
       font-size: 14px;
       cursor: default;
 
-      -webkit-transition: visibility 0.3s, -webkit-transform 0.3s;
-      transition: visibility 0.3s, transform 0.3s;
+      -webkit-transition: visibility 0.3s, -webkit-transform 0.3s, opacity 0.3s;
+      transition: visibility 0.3s, transform 0.3s, opacity 0.3s;
```

Comment:
opacity animation makes it nicer when the text inside the toast makes the `height > 100px`.